test/integration/replicaset/replicaset_test.go

```diff
@@ -166,7 +166,7 @@ func rmSimpleSetup(t *testing.T) (*httptest.Server, framework.CloseFunc, clients
 // running the RS controller to prevent the rc manager from creating new pods
 // rather than adopting the existing ones.
 func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podNum int) {
-	if err := wait.Poll(10*time.Second, 60*time.Second, func() (bool, error) {
+	if err := wait.Poll(100*time.Millisecond, 60*time.Second, func() (bool, error) {
```

Comment:
Could you move these values to package-level constants at the top of the file, like this?

```go
const (
	pollInterval = 100 * time.Millisecond
	pollTimeout  = 60 * time.Second
)
```

That way we can update them in one place.

Reply:
OK， thank a lot. I will fix it soon.